.github/workflows/test.yml

```diff
@@ -23,20 +23,4 @@ jobs:
     - name: Install dependencies and build
       run: npm ci
     - name: Run tests
-      run: npm test -- --exclude test/solc.js
-
-  # Solc tests run significantly faster in Node 18
```

Comment:
This was the reason tests were split. Ttests above used to run on v16, but it's no longer the case, so they can be merged.

Note we could also bump node-version to 20.x since 18 is EOL, but better to do that separately.